help/view-a-messages-edit-history.md

```diff
@@ -16,22 +16,12 @@ Organization administrators can
 
 {start_tabs}
 
-1. Click on the message's **EDITED** or **MOVED** label.
-
-{end_tabs}
-
-### Via the message menu (alternate method)
-
-{start_tabs}
-
-{!message-actions-menu.md!}
-
-1. Click **View edit history**.
+1. Click on the message's **EDITED** or **MOVED** label shown next to the user's name in the message feed.
 
 !!! warn ""
 
-    You will only see **View edit history** in the dropdown
-    menu if the message has been moved or edited, and
+    You will only see the **EDITED** or **MOVED** label
+    if the message has been moved or edited, and
     message edit history is enabled in the organization.
```

Comment:
I think this change is wrong -- my understanding is you'll see "EDITED" for messages that were either moved or edited without message edit history enabled, but you won't be able to click it to see edit history. Please test the behavior for both edited and moved messages and update this comment.

Reply:
> my understanding is you'll see "EDITED" for messages that were either moved or edited without message edit history enabled, but you won't be able to click it to see edit history. 

@timabbott This is true if message edit history is enabled and then gets disabled, but the **EDITED** and **MOVED** labels will disappear after refreshing/restarting the app.

Reply:
@drrosa , this behavior has changed in this latest PR #26161 . With the new behavior, the edited notice will always be there if the message is moved or edited, irrespective of the organization's settings. However, if it is enabled, then for moved messages, the notice will be moved instead of edited.

Reply:
@palashb01 Ah I see, thanks. Okay then we should change this documentation to something like:

```
    You will only see the **EDITED** or **MOVED** label if the message has been
    moved or edited. If message edit history is disabled in the organization,
    you won't be able to click these labels to see edit history.
```

Reply:
Done, thanks for the suggestion.